pom.xml

```diff
@@ -164,6 +165,25 @@
                 <artifactId>operator-framework</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <!--
+            regarding the okhttp explicit version
+            see https://github.com/fabric8io/kubernetes-client/issues/4290
+            -->
+            <dependency>
+                <groupId>com.squareup.okhttp3</groupId>
+                <artifactId>okhttp</artifactId>
+                <version>${okhttp.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.squareup.okhttp3</groupId>
+                <artifactId>logging-interceptor</artifactId>
+                <version>${okhttp.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.squareup.okhttp3</groupId>
+                <artifactId>mockwebserver</artifactId>
+                <version>${okhttp.version}</version>
```

Comment:
Well, technically is not shipped. Since it is transitively included only by this dependency in the core:

```
 <dependency>
      <groupId>io.fabric8</groupId>
      <artifactId>kubernetes-server-mock</artifactId>
      <scope>test</scope>
    </dependency>
```
What is test scoped. So won't be there in the final jar (that also true for the transitive dependencies). And for sake of simplicity would stick with the current approach. Unless there are other good arguments of course :)

Reply:
Thanks for the clarification. Didn't realize that it was added under `<dependencyManagement>`. No concerns then. 👍